dashboards/google-kubernetes-engine/gke-workloads-at-risk.json

```diff
@@ -18,103 +19,97 @@
         "widget": {
+          "title": "Workloads analyzed",
           "scorecard": {
             "blankView": {},
             "thresholds": [],
             "timeSeriesQuery": {
-              "timeSeriesQueryLanguage": "fetch k8s_container\n| metric 'kubernetes.io/container/cpu/core_usage_time'\n    | ${project_id}\n    | ${location}\n    | ${cluster_name}\n| filter resource.namespace_name !~ '(kube|istio|gatekeeper|gke|gmp|gke-gmp)-system|gke-managed-filestorecsi|gke-mcs'\n| align rate(5m)\n| every 5m\n| group_by\n    [resource.project_id, resource.location, resource.cluster_name, resource.namespace_name, metadata.system_labels.top_level_controller_name, metadata.system_labels.top_level_controller_type, resource.container_name],.max\n| filter value_core_usage_time_max > 0\n| row_count\n",
+              "timeSeriesQueryLanguage": "fetch k8s_container\n| metric 'kubernetes.io/container/cpu/core_usage_time'\n    | ${project_id}\n    | ${location}\n    | ${cluster_name}\n    | ${namespace_name}\n| filter resource.namespace_name !~ '(kube|istio|gatekeeper|gke|gmp|gke-gmp)-system|gke-managed-filestorecsi|gke-mcs'\n| align rate(5m)\n| every 5m\n| group_by\n    [resource.project_id, resource.location, resource.cluster_name, resource.namespace_name, metadata.system_labels.top_level_controller_name, metadata.system_labels.top_level_controller_type, resource.container_name],.max\n| filter value_core_usage_time_max > 0\n| row_count\n",
               "unitOverride": ""
             }
-          },
-          "title": "Workloads analyzed"
-        },
-        "width": 2,
-        "xPos": 8,
-        "yPos": 0
+          }
+        }
       },
       {
-        "height": 6,
+        "xPos": 0,
+        "yPos": 8,
+        "width": 48,
+        "height": 24,
         "widget": {
+          "title": "Best Effort - Memory/CPU not set",
           "collapsibleGroup": {
             "collapsed": false
-          },
-          "title": "Best Effort - Memory/CPU not set"
-        },
-        "width": 12,
-        "xPos": 0,
-        "yPos": 2
+          }
+        }
       },
       {
-        "height": 6,
+        "xPos": 0,
+        "yPos": 32,
+        "width": 48,
+        "height": 24,
         "widget": {
+          "title": "Burstable - Memory/CPU running above request",
           "collapsibleGroup": {
             "collapsed": false
-          },
-          "title": "Burstable - Memory/CPU running above request"
-        },
-        "width": 12,
-        "xPos": 0,
-        "yPos": 8
+          }
+        }
       },
       {
-        "height": 2,
+        "xPos": 0,
+        "yPos": 0,
+        "width": 8,
+        "height": 8,
         "widget": {
+          "title": "Memory not set",
           "scorecard": {
-            "thresholds": [
-              {
-                "color": "RED",
-                "direction": "ABOVE",
-                "label": "",
-                "value": 0
-              }
-            ],
+            "thresholds": [],
```

Comment:
Is it intentional that "Memory not set" lost its threshold? It used to be red if above 0

Reply:
this wasn't intentional @yqlu . I added the threshold back in